### PR TITLE
Proofread Swiss German readme

### PR DESCRIPTION
**Pull request by The-Compiler**
## Type of changes

- [ ] Bug fix
- [ ] New feature
- [x] Documentation / docstrings
- [ ] Tests
- [ ] Other

## Checklist

- [ ] I've run the latest [black](https://github.com/psf/black) with default args on new code.
- [ ] I've updated CHANGELOG.md and CONTRIBUTORS.md where appropriate.
- [ ] I've added tests for new code.
- [x] I accept that @willmcgugan may be pedantic in the code review.

## Description

Less typos, yay! There's no official written version of Swiss German, and this version reads horrible to me to be honest (but I know the author, so I can say so :laughing:), but I only fixed the obvious typos.

### CODE REVIEW

**Review by das-g — CHANGES_REQUESTED** [review]
These changes improve the translation and its correctness significantly.

Only some minor nitpicks:

**Comment by willmcgugan** [comment]
Let me know when this is ready to merge. I'm afraid I can't contribute!

File: README.de-ch.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ Lueg was [anderi öber Rich säged](https://www.willmcgugan.com/blog/pages/post/
...
 
-Rich funktioniert mit Linux, OSX ond Windows. True color / emoji funktioniert mit em neue Windows Törminäl, s klassische Törminäl isch of 16 Farbe limitiärt. Rich brucht Pzthon 3.6.1 oder neuer.
+Rich funktioniert mit Linux, OSX ond Windows. True color / emoji funktioniert mit em neue Windows Törminäl, s klassische Törminäl isch of 16 Farbe limitiärt. Rich brucht Python 3.6.1 oder neuer.
 
 Rich funktioniert mit [Jupyter notebooks](https://jupyter.org/) ohni irgendwelchä zuäsätzloche konfiguration.
 
 ## Installation
 
-Installation mit `pip` oder mit dim liäblings PyPi Päckli-Mangager.
+Installation mit `pip` oder mit dim liäblings PyPI Päckli-Mangager.
```
Comment (Chive) [inline README.de-ch.md:40]:
```suggestion
Installation mit `pip` oder mit dim liäblings PyPI Päckli-Manager.
```

**Comment by ubaumann** [comment]
No rules in Swiss German so everything is fine and many typos are found (sorry for that).
All changes are fine for me

**Comment by The-Compiler** [comment]
@willmcgugan Should be ready now from my POV!

**Comment by willmcgugan** [comment]
Fantastisch! Thanks all.

**Pull request merged**